Description: doc: fix old dpdk-nic-bind.py references

dpdk-nic-bind.py script has been renamed to dpdk-devbind.py,
but some references to the old script have remained.
This commit completes the renaming.

Fixes: a5d7a3f77ddc ("unify tools naming")

Signed-off-by: Pablo de Lara <pablo.de.lara.guarch@intel.com>

Origin: Upstream, http://dpdk.org/dev/patchwork/patch/15057/
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Last-Update: 2016-08-03

diff --git a/doc/guides/howto/flow_bifurcation.rst b/doc/guides/howto/flow_bifurcation.rst
index a1c6262..0d7226a 100644
--- a/doc/guides/howto/flow_bifurcation.rst
+++ b/doc/guides/howto/flow_bifurcation.rst
@@ -119,8 +119,8 @@ The typical procedure to achieve this is as follows:
    .. code-block:: console
 
        modprobe vfio-pci
-       dpdk_nic_bind.py -b vfio-pci 01:10.0
-       dpdk_nic_bind.py -b vfio-pci 01:10.1
+       dpdk-devbind.py -b vfio-pci 01:10.0
+       dpdk-devbind.py -b vfio-pci 01:10.1
 
 #. Run a DPDK application on the VFs:
 
@@ -279,8 +279,8 @@ The typical procedure to achieve this is as follows:
    .. code-block:: console
 
        modprobe vfio-pci
-       dpdk_nic_bind.py -b vfio-pci 01:10.0
-       dpdk_nic_bind.py -b vfio-pci 01:10.1
+       dpdk-devbind.py -b vfio-pci 01:10.0
+       dpdk-devbind.py -b vfio-pci 01:10.1
 
 #. run DPDK application on VFs:
 
diff --git a/doc/guides/howto/lm_bond_virtio_sriov.rst b/doc/guides/howto/lm_bond_virtio_sriov.rst
index 49666f1..9bc3450 100644
--- a/doc/guides/howto/lm_bond_virtio_sriov.rst
+++ b/doc/guides/howto/lm_bond_virtio_sriov.rst
@@ -613,17 +613,17 @@ Set up DPDK in the Virtual Machine
    cat  /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
 
    ifconfig -a
-   /root/dpdk/tools/dpdk_nic_bind.py --status
+   /root/dpdk/tools/dpdk-devbind.py --status
 
    rmmod virtio-pci ixgbevf
 
    modprobe uio
    insmod /root/dpdk/x86_64-default-linuxapp-gcc/kmod/igb_uio.ko
 
-   /root/dpdk/tools/dpdk_nic_bind.py -b igb_uio 0000:00:03.0
-   /root/dpdk/tools/dpdk_nic_bind.py -b igb_uio 0000:00:04.0
+   /root/dpdk/tools/dpdk-devbind.py -b igb_uio 0000:00:03.0
+   /root/dpdk/tools/dpdk-devbind.py -b igb_uio 0000:00:04.0
 
-   /root/dpdk/tools/dpdk_nic_bind.py --status
+   /root/dpdk/tools/dpdk-devbind.py --status
 
 run_testpmd_bonding_in_vm.sh
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
diff --git a/doc/guides/howto/lm_virtio_vhost_user.rst b/doc/guides/howto/lm_virtio_vhost_user.rst
index fad1f2a..4937781 100644
--- a/doc/guides/howto/lm_virtio_vhost_user.rst
+++ b/doc/guides/howto/lm_virtio_vhost_user.rst
@@ -91,14 +91,14 @@ For Fortville NIC.
 .. code-block:: console
 
    cd /root/dpdk/tools
-   ./dpdk_nic_bind.py -b igb_uio 0000:02:00.0
+   ./dpdk-devbind.py -b igb_uio 0000:02:00.0
 
 For Niantic NIC.
 
 .. code-block:: console
 
    cd /root/dpdk/tools
-   ./dpdk_nic_bind.py -b igb_uio 0000:09:00.0
+   ./dpdk-devbind.py -b igb_uio 0000:09:00.0
 
 On host_server_1: Terminal 3
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@@ -172,14 +172,14 @@ For Fortville NIC.
 .. code-block:: console
 
    cd /root/dpdk/tools
-   ./dpdk_nic_bind.py -b igb_uio 0000:03:00.0
+   ./dpdk-devbind.py -b igb_uio 0000:03:00.0
 
 For Niantic NIC.
 
 .. code-block:: console
 
    cd /root/dpdk/tools
-   ./dpdk_nic_bind.py -b igb_uio 0000:06:00.0
+   ./dpdk-devbind.py -b igb_uio 0000:06:00.0
 
 On host_server_2: Terminal 3
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@@ -444,17 +444,17 @@ setup_dpdk_virtio_in_vm.sh
    cat  /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
 
    ifconfig -a
-   /root/dpdk/tools/dpdk_nic_bind.py --status
+   /root/dpdk/tools/dpdk-devbind.py --status
 
    rmmod virtio-pci
 
    modprobe uio
    insmod /root/dpdk/x86_64-default-linuxapp-gcc/kmod/igb_uio.ko
 
-   /root/dpdk/tools/dpdk_nic_bind.py -b igb_uio 0000:00:03.0
-   /root/dpdk/tools/dpdk_nic_bind.py -b igb_uio 0000:00:04.0
+   /root/dpdk/tools/dpdk-devbind.py -b igb_uio 0000:00:03.0
+   /root/dpdk/tools/dpdk-devbind.py -b igb_uio 0000:00:04.0
 
-   /root/dpdk/tools/dpdk_nic_bind.py --status
+   /root/dpdk/tools/dpdk-devbind.py --status
 
 run_testpmd_in_vm.sh
 ~~~~~~~~~~~~~~~~~~~~
